vms/avm/wallet_service.go

```diff
@@ -31,27 +31,26 @@ type WalletService struct {
 }
 
 func (w *WalletService) decided(txID ids.ID) {
```

Comment:
This is called during `block.Accept` - so we must support holding the context lock here.